### PR TITLE
Adding CIR-KIT utilities to documentation index for distro

### PR DESCRIPTION
**Pull request by MoriKen254**
I'd like to some utility packages for our autonomous robot to be indexed and documented on ros.org.

### CODE REVIEW

**Comment by MoriKen254** [comment]
Could you tell me when the index will be actually active after this pull request is merged?

**Comment by 130s** [comment]
@MoriKen254 [This thread](http://answers.ros.org/question/230803/how-can-i-check-the-status-of-my-package-submission/) should be of your interest.

File: indigo/distribution.yaml [hunk 1]
```diff
@@ -5738,6 +5748,16 @@ repositories:
       url: https://github.com/orocos-toolchain/log4cpp.git
       version: toolchain-2.8
     status: maintained
+  lower_step_detector:
```
Comment (dirk-thomas) [inline indigo/distribution.yaml]: Shouldn't this be called `remote_monitor`?

**Comment by MoriKen254** [comment]
@dirk-thomas Excuse me, you are right. I fixed the typo.

@130s Thank you. However http://build.ros.org/ doesn't show on a browser because of proxy error now... Is this supposed to be fix?

**Pull request merged**